Sources/SwiftStylable/Classes/Style/StylableComponents/STStackView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  STStackView.swift
+//  Pods-SwiftStylableExample
+//
+//  Created by Rens Wijnmalen on 23/03/2020.
+//
+
+import UIKit
+
+@IBDesignable open class STStackView: UIStackView {
+
+    private var _dimension:String?
+    
+    // -----------------------------------------------------------------------------------------------------------------------
+    //
+    // MARK: - Initializers & deinit
+    //
+    // -----------------------------------------------------------------------------------------------------------------------
+    
+    public init() {
+        super.init(frame: CGRect.zero) 
+        NotificationCenter.default.addObserver(self, selector: #selector(STStackView.stylesDidUpdate(_:)), name: STYLES_DID_UPDATE, object: nil)
+    }
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        NotificationCenter.default.addObserver(self, selector: #selector(STStackView.stylesDidUpdate(_:)), name: STYLES_DID_UPDATE, object: nil)
+    }
+    
+    required public init(coder: NSCoder) {
+        super.init(coder: coder)
+        NotificationCenter.default.addObserver(self, selector: #selector(STStackView.stylesDidUpdate(_:)), name: STYLES_DID_UPDATE, object: nil)
+    }
+    
+    open override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(STStackView.stylesDidUpdate(_:)), name: STYLES_DID_UPDATE, object: nil)
+    }
+    deinit {
+        NotificationCenter.default.removeObserver(self)
+    }
+    
+    // -----------------------------------------------------------------------------------------------------------------------
+    //
+    // MARK: - Computed properties
+    //
+    // -----------------------------------------------------------------------------------------------------------------------
+    
+    @IBInspectable open var dimension:String? {
```

Comment:
Het is best mogelijk dat ik dit in ander components gemist heb, maar ik denk niet dat dit zo moet werken. Dimensions zijn eigenlijk onderdeel van de style, dus je set een dimension nooit direct. Je haalt m altijd uit de style.

(Ook is dimension hier een beetje generiek, welke dimensie gaat het om?)